x/auction/keeper/auctions_test.go

```diff
@@ -1,470 +1,470 @@
-	_, addrs := app.GeneratePrivKeyAddressPairs(1)
-	buyer := addrs[0]
-	sellerModName := cdptypes.LiquidatorMacc
-	sellerAddr := authtypes.NewModuleAddress(sellerModName)
-
-	tApp := app.NewTestApp()
-
-	sellerAcc := authtypes.NewEmptyModuleAccount(sellerModName, authtypes.Burner) // forward auctions burn proceeds
-	// require.NoError(t, sellerAcc.SetCoins(cs(c("token1", 100), c("token2", 100))))
-	tApp.InitializeFromGenesisStates(
-		// NewAuthGenStateFromAccs(authtypes.GenesisAccounts{
-		// 	authtypes.NewBaseAccount(buyer, cs(c("token1", 100), c("token2", 100)), nil, 0, 0),
-		// 	sellerAcc,
-		// }),
-		app.NewAuthGenState(tApp, []sdk.AccAddress{buyer}, []sdk.Coins{cs(c("token1", 100), c("token2", 100))}),
-		app.NewAuthGenStateModAcc(tApp, []*authtypes.ModuleAccount{sellerAcc}),
-	)
-	ctx := tApp.NewContext(false, tmproto.Header{})
-	require.NoError(t, simapp.FundAccount(tApp.BankKeeper, ctx, sellerAddr, cs(c("token1", 100), c("token2", 100))))
-
-	keeper := tApp.GetAuctionKeeper()
-
-	// Create an auction (lot: 20 token1, initialBid: 0 token2)
-	id, err := keeper.StartSurplusAuction(ctx, sellerModName, c("token1", 20), "token2") // lot, bid denom
-	require.NoError(t, err)
-
-	// Attempt to close the auction before EndTime
-	require.Error(t, keeper.CloseAuction(ctx, id))
-
-	// Attempt to close auction that does not exist
-	require.Error(t, keeper.CloseAuction(ctx, 999))
-}
-
-func TestCloseExpiredAuctions(t *testing.T) {
-	// Set up
-	_, addrs := app.GeneratePrivKeyAddressPairs(1)
-	buyer := addrs[0]
-	sellerModName := "liquidator"
-	sellerAddr := authtypes.NewModuleAddress(sellerModName)
-
-	tApp := app.NewTestApp()
-
-	sellerAcc := authtypes.NewEmptyModuleAccount(sellerModName, authtypes.Burner) // forward auctions burn proceeds
-	// require.NoError(t, sellerAcc.SetCoins(cs(c("token1", 100), c("token2", 100))))
-	tApp.InitializeFromGenesisStates(
-		// NewAuthGenStateFromAccs(authtypes.GenesisAccounts{
-		// 	authtypes.NewBaseAccount(buyer, cs(c("token1", 100), c("token2", 100)), nil, 0, 0),
-		// 	sellerAcc,
-		// }),
-		app.NewAuthGenState(tApp, []sdk.AccAddress{buyer}, []sdk.Coins{cs(c("token1", 100), c("token2", 100))}),
-		app.NewAuthGenStateModAcc(tApp, []*authtypes.ModuleAccount{sellerAcc}),
-	)
-	ctx := tApp.NewContext(false, tmproto.Header{})
-	require.NoError(t, simapp.FundAccount(tApp.BankKeeper, ctx, sellerAddr, cs(c("token1", 100), c("token2", 100))))
-
-	keeper := tApp.GetAuctionKeeper()
-
-	// Start auction 1
-	_, err := keeper.StartSurplusAuction(ctx, sellerModName, c("token1", 20), "token2") // lot, bid denom
-	require.NoError(t, err)
-
-	// Start auction 2
-	_, err = keeper.StartSurplusAuction(ctx, sellerModName, c("token1", 20), "token2") // lot, bid denom
-	require.NoError(t, err)
-
-	// Fast forward the block time
-	ctx = ctx.WithBlockTime(ctx.BlockTime().Add(auctiontypes.DefaultMaxAuctionDuration).Add(1))
-
-	// Close expired auctions
-	err = keeper.CloseExpiredAuctions(ctx)
-	require.NoError(t, err)
-}
+// import (
```

Comment:
@jununifi 
Can you tell me why you are commenting out here?

Reply:
Same here